ui/i18n/messages-ar.json

```diff
@@ -199,7 +199,7 @@
   "sortSelect.reverseOrder": "Reverse sort order",
   "tableSettings.columnSettings": "Columns Settings",
   "tableSettings.columnSettings.close": "Close",
-  "tableSettings.columnSettings.reset": "Reset",
```

Comment:
I think you meant to remove `tableSettings.columnSettings.close` and not `tableSettings.columnSettings.reset`.

In below locale, it is `tableSettings.columnSettings.close` that you remove.